cli/src/bin/phylum.rs

```diff
@@ -203,6 +193,6 @@ async fn main() {
             ),
         },
         Ok(CommandValue::Code(code)) => code.exit(),
-        Err(error) => exit_error(error.into(), "Execution failed"),
+        Err(error) => exit_fail(error, ExitCode::Generic),
```

Comment:
What's the motivation behind this change? I'd assume it just didn't look good with the new error messages?

Reply:
There are two changes here:

1. `Error: Execution failed caused by:` is replaced with `Error:`. This was mostly to reduce clutter in the output. I don't think the longer message provided any more information.
2. The `exit_error` function goes away. This was motivated by the first change since `exit_fail` already prints `Error:`, so it was easy to combine the 2 functions.